app/models/user.rb

```diff
@@ -112,7 +112,7 @@ def send_request_to_patron_creator_service
         'pcode3' => pcode3,
         'pcode4' => pcode4
       },
-      'barcodes' => [self.assign_barcode.to_s],
+      'barcodes' => [self.barcode.present? ? self.barcode : self.assign_barcode.to_s],
```

Comment:
Seems like your adding this logic to use this method on an existing user. Is that correct?

Reply:
@nat425 if an admin enters a barcode for a patron when they are creating that patron through the admin dashboard, then we want to use that barcode in send_request_to_patron_creator_service, otherwise we want to assign a new barcode.

Previously, we were assigning barcodes within the `send_request_to_patron_creator_service` method for all patrons.